app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -31,4 +31,22 @@ def raw_delete_volume
   def validate_delete_volume
     {:available => true, :message => nil}
   end
+
+  # ================= edit  ================
+
+  # override the function in manageiq\app\models\cloud_volume\operations.rb
+  def validate_update_volume
+    {:available => true, :message => nil}
+  end
+
+  def raw_update_volume(options = {})
+    ems = ext_management_system
+    vol_to_update = ems.autosde_client.VolumeUpdate(
```

Comment:
This isn't a volume to update, what updates we want to make. An appropriate name could be `volume_update`, `update_details`, `things_to_change`, etc..

Reply:
agree. changed to update_details